contracts/astroport/oracle/src/mock_querier.rs

```diff
@@ -102,7 +125,7 @@ impl WasmMockQuerier {
                         ),
                         _ => panic!("DO NOT ENTER HERE"),
                     }
-                } else {
+                } else if contract_addr == "pair" {
```

Comment:
Please move "pair" into constant and use it here.

Reply:
I really wouldn't waste time on it atm. this code is taken as is from the origin — astroport dudes